nvm.sh

```diff
@@ -2901,7 +2911,18 @@ nvm() {
           -s)
             shift # consume "-s"
             nobinary=1
```

Comment:
see above; you'll need to declare `nosource` the same way `nobinary` is declared.